pages/docs/connect/admin_ui.html

```diff
@@ -43,6 +38,12 @@
 ### Cluster Tab
 Provides an overview of nodes in the cluster including details such as load, CPU usage, shards on the node and hostname.
 
+### Monitoring Tab
+Gives an overview of the average number and speed of CrateDB queries (`SELECT`, `INSERT`, `UPDATE` and `DELETE`). Please note that this feature is only available to the `Enterprise` customers. 
```

Comment:
"only available to the `Enterprise` customers" -> "only available in the Enterprise Edition of CrateDB."

could maybe link "Enterprise Edition" to https://crate.io/pricing/